### PR TITLE
JS2 week 5

### PR DESCRIPTION
**Pull request by melanieet**
# Your Details

Your Name: **Melanie**
Your City: **London**
Your Slack Name: **Melanie**

# Homework Details

Module: JS2
Week: 5

### CODE REVIEW

**Comment by melanieet** [comment]
I still need to work on the final exercise.

File: week-5/Homework/mandatory/2-exercises/exercises.js [hunk 1]
```diff
@@ -13,8 +13,17 @@
  *      .....
  * </div>
  */
+
 function exerciseOne(arrayOfPeople) {
-  let content = document.querySelector("#content");
+    let content = document.querySelector("#content");
+    arrayOfPeople.forEach((person) => {
```
Comment (amellie) [inline week-5/Homework/mandatory/2-exercises/exercises.js:19]: well done!

File: week-5/Homework/mandatory/2-exercises/exercises.js [hunk 2]
```diff
@@ -24,8 +33,17 @@ function exerciseOne(arrayOfPeople) {
...
  *
  */
+
 function exerciseTwo(shopping) {
-  //Write your code in here
+    let content = document.querySelector("#content");
+    let ul = document.createElement("ul");
+    content.appendChild(ul);
+
+    shopping.forEach((item) => {
```
Comment (amellie) [inline week-5/Homework/mandatory/2-exercises/exercises.js:42]: thats great. it shows that you clearly understand how to traverse through the array and adding ul and li tags to the content

File: week-5/Homework/mandatory/2-exercises/exercises.js [hunk 3]
```diff
@@ -58,7 +76,36 @@ function exerciseTwo(shopping) {
...
+        img.setAttribute("height", "277");
+        p.innerText = book.title + " " + "-" + " " + book.author;
+        li.appendChild(p);
+        li.appendChild(img);
+        ul.appendChild(li);
+
+        if (book.title == "The Design of Everyday Things") {
+            img.src =
+                "https://images-na.ssl-images-amazon.com/images/I/41bWcNdTGmL._SX330_BO1,204,203,200_.jpg";
+            li.style.backgroundColor = "red";
```
Comment (amellie) [inline week-5/Homework/mandatory/2-exercises/exercises.js:98]: this definitely works perfectly. however, there is a more efficient way to do this by looking at the `alreadyRead` property in the `books` object

**Pull request closed without merging**